examples/mouse_to_tile.rs

```diff
@@ -238,9 +265,11 @@ fn swap_map_type(
                 }
             }
 
+            *map_transform = get_tilemap_center_transform(&map_size, &grid_size, &map_type, 0.0);
+
             for (tile_pos, mut tile_label_transform) in tile_label_q.iter_mut() {
                 let tile_center = tile_pos.center_in_world(&grid_size, &map_type).extend(1.0);
-                *tile_label_transform = *map_transform * Transform::from_translation(tile_center);
+                *tile_label_transform = Transform::from_translation(tile_center);
```

Comment:
See other comments: we need to check to make sure that this still works when the map is panned